src/transformer/cli.js

```diff
@@ -53,28 +53,69 @@ export default async ({name, yargsOptions, callback}) => {
 		process.exit(-1);
 	}
 
-	const spinner = ora('Transforming records').start();
+	const spinner = ora(`Transforming${args.validate ? ' and validating' : ''}${args.fix ? ' and fixing' : ''} records`).start();
+
 	const options = {
 		stream: fs.createReadStream(args.file),
-		args,
-		spinner,
-		handleRecordsOutput
+		args
 	};
-	await callback(options);
 
-	function handleRecordsOutput(records) {
+	const TransformClient = callback(options);
+	let succesRecordArray = [];
+	let failedRecordsArray = [];
+	let pendingPromises = [];
```

Comment:
- Use const
- Declare in the block it's actually used
- Change name to TransformEmitter or something like that to better reflect the functionality.
- Fix typo `succesRecordArray` to `successRecordArray`